samples/ClientSample/ClientSample.csproj

```diff
@@ -1,11 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It actually doesn't work well. I'm not sure why we made it web in the first place. The default is launching IIS express which makes 0 sense here.